airflow-core/src/airflow/ui/.gitignore

```diff
@@ -0,0 +1 @@
+openapi.merged.json
```

Comment:
The merged output of `openapi-merge-cli` -- since both sources and the generated code is committed, we don't want this too.